### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dslzuha**
## Bug

Fixes: Link_to_issue  
Regression: Yes/No  
* Last working version:   
* How are we preventing it in future:   

## Fix

Details: Details_about_the_fix  

## Testing/Validation

Tests Added: Yes/No  
Reason for not adding tests:  
Validation:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
     `.\runTests.ps1`
 
 
-> In case you have build issues try cleaning the local repository using `git clean -xdf` and retry steps 3 and 4.
```
Comment (nkolev92) [inline README.md:53]: I actually think `you` reads better in this case.

Comment (nkolev92) [reply README.md:53]: Do you have any otther proposed changes or should I clsoe this PR?

**Comment by nkolev92** [comment]
Closing per the above comment.

**Comment by rrelyea** [comment]
You seem to be opening a number of PRs in our repo without significant changes. Perhaps you are learning to use GitHub and did not intend to send them to us. Please be aware that other people may be working in repos you do not own, so if you are testing, we suggest you create your own repo and follow steps similar to this: https://guides.github.com/activities/hello-world/

If you are hoping to contribute to nuget.client repro, please make sure to read https://github.com/NuGet/NuGet.Client/blob/dev/CONTRIBUTING.md guidelines and start with step 1 (open an issue), not step 5 (open a PR).

**Pull request closed without merging**